torch/autograd/forward_ad.py

```diff
@@ -99,11 +102,18 @@ def unpack_dual(tensor, *, level=None):
     return UnpackedDualTensor(primal, dual)
 
 class dual_level(_DecoratorContextManager):
-    r"""Context-manager that controls the current forward ad level. It
-    appropriately enters and exit the dual level.
+    r"""Context-manager that enables forward ad. All forward AD computation must
+    be performed in a ``dual_level`` context.
 
-    This function also updates the current level that is used by default
-    by the other functions in this API.
+    .. Note::
+
+        The ``dual_level`` context appropriately enters and exit the dual level to
+        controls the current forward ad level, which is used by default by the other
+        functions in this API.
+
+        We currently don't plan to support nested ``dual_level`` contexts, however, so
+        only a single forward ad level is supported. To compute higher-order
+        forward grads, one can use `functorch <https://github.com/pytorch/functorch>`__.
```

Comment:
Should this point somewhere more specifically within function? Maybe the jvp function?

Reply:
Hmm I couldn't find the reference for jvp within functorch actually, maybe it doesn't exist yet? cc @zou3519

Reply:
JVP function exists, the docs for it don't show up yet on the website because the docs show the functorch that is compatible with PyTorch 1.10 and jvp isn't there...

Reply:
TODO(@zou3519: add functorch.jvp to functorch readme, send link here)

Reply:
Let's link this for now: https://github.com/pytorch/functorch#jvp

When the functorch docs website goes live sometime in the next month (I'm not sure when this will be) I can link to the website instead of this.

Reply:
Sounds good!